README.md

```diff
@@ -245,14 +246,14 @@ echo "Hello World" | sttr base64-encode | sttr md5
 - [x] **interactive** - Use sttr in interactive mode
 - [x] **version** - Print the version of sttr
 - [x] **zeropad** - Pad a number with zeros
-- [x] **and adding more....**
+- [x] **and adding more...**
 
 # Featured On
 
-These are the few locations where `sttr` was highlighted, many thanks to all of you. 
-Please feel free to add any blogs/videos you may have made that discuss `sttr` to the list.
+These are the few places where `sttr` got featured, many thanks to everyone.
+If you have talked about `sttr` in your blog/video feel to add that to the list.
```

Comment:
I'm doubtful about what happened here.

I didn't change these lines on purpose.

It looks like broken conflict resolution.

I'm sorry @abhimanyu003

Reply:
Yes, it looks like that. I will fix this. No worries on this. :)